sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -260,13 +276,20 @@ class AioHttpTransportResponse(AsyncHttpResponse):
     :param block_size: block size of data sent over connection.
     :type block_size: int
     """
-    def __init__(self, request: HttpRequest, aiohttp_response: aiohttp.ClientResponse, block_size=None) -> None:
+    def __init__(self, request, aiohttp_response, block_size=None):
+        # type: (HttpRequest, ClientResponse, int) -> None
```

Comment:
That's a type change here. Instead, do:
```from typing import TYPE_CHECKING
if TYPE_CHECKING:
    import aiohttp

and use `"aiohttp.ClientResponse"` as a string (instead of using the comment syntax)